### PR TITLE
Make `./mach rustup` use the latest nightly rather than master.

### PR DESCRIPTION
**Pull request by SimonSapin**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/16836)

### CODE REVIEW

**Comment by SimonSapin** [comment]
r? @nox

(This will also affect https://travis-ci.org/servo/servo-with-rust-nightly which now uses `./mach rustup`.)

File: python/servo/devenv_commands.py [hunk 1]
```diff
@@ -228,9 +228,20 @@ def grep(self, params):
     @Command('rustup',
              description='Update the Rust version to latest master',
```
Comment (cbrewster) [inline python/servo/devenv_commands.py]: Should this say `Update the Rust version to latest nightly` now?

Comment (SimonSapin) [inline python/servo/devenv_commands.py]: Yes. Fixed.

**Comment by SimonSapin** [comment]
Accidentally included a rustup in the previous commit.

@bors-servo r=nox

**Pull request merged**